plop-templates/Component/Component.stories.tsx.hbs

```diff
@@ -7,10 +7,13 @@ import { {{pascalCase name}}, Props } from './{{pascalCase name}}';
 export default {
 	title: 'Example/{{pascalCase name}}',
 	component: {{pascalCase name}},
+	parameters: {
+    badges: [BADGE.BETA],
+  },
```

Comment:
I don't know why the indentation in these files look so weird in github PRs but this file looks fine in VSCode.

Reply:
That was it! Thanks! ❤️